tests/dummy/app/styles/app.scss

```diff
@@ -40,29 +39,29 @@ h3 {
   background-image: linear-gradient(-134deg, #3023AE 0%, #A56DD7 100%);
   -webkit-display: flex;
   display: flex;
-  @include justify-content(center);
-  @include align-items(center);
-  @include flex-direction(column);
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```

Comment:
I'm making the assumption that ember developer are using up-to-date browsers, which don't need vendor prefixes. If we want to make our dummy app more cross-browser friendly, we could use [ember-cli-postcss](https://github.com/jeffjewiss/ember-cli-postcss). I think that's outside the scope of this PR.